pages/common/lex.md

```diff
@@ -0,0 +1,16 @@
+# lex
+
+> Lexical analyser generator.
+> Given a lexical analyser, generates C code implementing it.
+
+- Generate analyser from Lex file:
```

Comment:
Perhaps `an analyser` and `a Lex file` would read better here?

Also, you've put `lex` with a capital `L` here, but used a lowercase `l` further down. Is this intentional?

Reply:
Both good points; addressed in new commit. :)